api/types/event/snapshot.proto

```diff
@@ -0,0 +1,17 @@
+syntax = "proto3";
+
+package containerd.v1.types;
```

Comment:
Should we declare these types in the package that emits the events, along with the service?